routers/api/v1/api.go

```diff
@@ -1,4 +1,5 @@
 // Copyright 2015 The Gogs Authors. All rights reserved.
+// Copyright 2016 The Gitea Authors. All rights reserved.
```

Comment:
2018

Reply:
I think maybe 2016 because the file might have been modified then.

Reply:
OK

Reply:
@techknowlogick @zeripath yes. If I explicitly know the file is modified on some year I will change to that year, but will change it to 2018.